als_driver.c

```diff
@@ -0,0 +1,22 @@
+#define ALS_EN_GPIO 11
+#define ALS_READ_ADDR 0xA110FF30
+
+
+#include <stdio.h>
```

Comment:
and what is stdio doing here - our embedded system doesn't support stdio.h

Reply:
this is for emulation, not for the embedded system.  Once the rest of the team adds their code and tests it, then all of the printfs and stdio will be removed, and replace with the embedded code

Reply:
ok, I understand.